framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/ViewFeatures/AbpRazorCompiledItemFeatureProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.AspNetCore.Mvc.Razor.Compilation;
+using Microsoft.Extensions.DependencyInjection;
+using Volo.Abp.VirtualFileSystem;
+
+namespace Volo.Abp.AspNetCore.Mvc.ViewFeatures
+{
+    /// <summary>
+    /// The code for this class comes from
+    /// https://github.com/dotnet/aspnetcore/blob/master/src/Mvc/Mvc.Razor/src/ApplicationParts/RazorCompiledItemFeatureProvider.cs
+    /// </summary>
+    public class AbpRazorCompiledItemFeatureProvider : IApplicationFeatureProvider<ViewsFeature>
```

Comment:
This class does not seem to be needed, because razor runtime compilation will use the virtual file system to compare whether the page needs to be recompiled.